### ISSUE TITLE
Images with links don't work

### ISSUE THREAD

**Issue by AndyCErnst** [prismicio/scala-kit#45]
The `hyperlink` field should be populated when an image has a link but it comes through as `None` making it impossible to have an anchor wrapper around an image

```
          case StructuredText.Block.Image(view, hyperlink, label, dir) => {
            val linkbody = hyperlink match {
              case Some(link: DocumentLink) => s"""<a href="$linkResolver(link)">${view.asHtml}</a>"""
```

line 325 in io.prismic.fragments.package

**Comment by dohzya** [comment]
Hi,

I'm not sure to understand, when do you get a `None` (comparing to [this test](https://github.com/prismicio/scala-kit/blob/master/src/test/scala/FragmentSpec.scala#L387))?

**Comment by AndyCErnst** [comment]
It looks like it may have been fixed in commit 4331dce85bdf8377858d0dfaf1fdec25d3df974b but my team can't go to the latest version until the netty functionality in https://github.com/prismicio/scala-kit/pull/41 is pulled in. So it's a me problem. Closing this issue.
